app/core/src/test/java/com/fsck/k9/helper/ListUnsubscribeHelperTest.kt

```diff
@@ -0,0 +1,89 @@
+package com.fsck.k9.helper
+
+import com.fsck.k9.K9RobolectricTest
+import com.fsck.k9.mail.internet.MimeMessage
+import org.junit.Assert
+import org.junit.Test
+
+class ListUnsubscribeHelperTest : K9RobolectricTest() {
+    @Test
+    fun getListUnsubscribeUrl_shouldAcceptMailto() {
```

Comment:
If you want to, you can use slightly easier to read function names. Kotlin allows using spaces and other characters. See e.g.

https://github.com/k9mail/k-9/blob/e1ff582cac7191254dc00d61ebfcab63bfce8011/mail/protocols/smtp/src/test/java/com/fsck/k9/mail/transport/smtp/SmtpResponseParserTest.kt#L623